themes/bootstrap3/templates/RecordTab/holdingsils/boundwith.phtml

```diff
@@ -0,0 +1,18 @@
+<div class="bound-with-records">
+    <h4>
+        <?=$this->transEsc(
+            $callNumberCount < 2 ? 'bound_with_heading_one_callnumber' : 'bound_with_heading_multiple_callnumbers', 
+            ['%%callnumber%%' => $this->escapeHtml($callNumber)])
+        ?>
+    </h4>
+    <p><?=$this->translate('bound_with_intro', ['%%title%%' => $this->escapeHtml($this->truncate($driver->getTitle(), 50))])?></p>
+    <ol>
+        <?php foreach ($boundWithRecords as $record): ?>
+            <li class="<?=($record['bibId']==$driver->getUniqueId()) ? 'direct-item' : 'bound-with-item'?>">
+                <a href="<?=$this->escapeHtmlAttr($record['bibId'])?>">
+                    <?=$this->escapeHtml($this->truncate($record['title'], 45)) ?>
```

Comment:
This truncation is only necessary because the list of titles is displayed in the second column of this items table, taking up half its width, and approx 1/3 of total width including the sidebar column.  

- I would prefer it to be full-width on the table, but couldn't think of a way to do that while preserving the `th` / `td` structure for accessibility.  
    - If I simply put this info in a separate `<tr>` following the main one for this item, it messes up the alternating color CSS.  
    - I could alternately just adjust the column withs to something like 25%/75%, but the table has fixed layout and it might look odd for it to be relative, especially if there are other item tables on the same record (i.e. for other locations) that continue to have fixed width.  
    - I can't just display the list outside of the table itself, because it's semantically relevant to the specific item -- not to the bib/instance record.  
- I also suspect individual institutions will want to change that specific number.  I could move that to a config file but don't see a good design pattern for using config (beyond the base config.ini) within templates.  Alternately I could do the truncation earlier in `Folio.ini`.  But again I'd prefer to just have a wider column.
Ideas welcome.

Reply:
I would be in favor of changing the table layout away from a fixed 50/50%. It might be best to give these tables a class like `holdings-table` since the fixed layout is coming from the Bootstrap framework.

Reply:
As far as truncating goes, make sure to test it on the sandal theme for landing on a number. It has the largest font and padding.

Reply:
@crhallberg I've submitted a separate PR #3096 to adjust the columns on all holdings tab tables to 75/25 as we discussed in the zoom on Friday.  I've changed the truncation here to a level that seems to work fine (with a bit of margin to spare) even on Sandal.  Where it wraps on mobile widths it still doesn't seem to exceed 2 lines each.